python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -654,7 +654,7 @@ def delete_events_for_run(self, conn: Connection, run_id: str) -> None:
         )
 
         removed_asset_keys = [
-            AssetKey.from_db_string(row[0])
+            AssetKey.from_db_string(row[0])  # type: ignore
```

Comment:
latest version of sqlalchemy types does not like indexing into rows (it thinks they are some kind of fancy tuple, even though indexing works fine still)

Reply:
Wow that seems like a bug in the types. Hopefully it goes away with a future update.